docs/csharp/cs-dev-kit-faq.md

```diff
@@ -215,6 +215,12 @@ If you are encountering issues with Test Explorer, you can enable diagnostic log
 
 This information will help in diagnosing and resolving the issue more efficiently.
 
+### Known Issues
+
+- Using xUnit v3 with package Microsoft.Testing.Platform on macOS can cause test discovery and debugging issues in the Test Explorer for C# Devkit and Visual Studio.
```

Comment:
We probably do not need to mention VS here since this doc is tailored to C# Dev Kit specifically.